### DISCUSSION TITLE
Long term support strategy

### DISCUSSION THREAD

**Discussion by steven-bellock** [DMTF/libspdm#1572] [General]
We intend for libspdm 3.0, to be released at the end of Q1 2023, to be a long term support release. But we still need to define
- What will be supported.
- The duration of support.
- Who provides support.

Drawing from, for example, the [Ubuntu release cycle](https://ubuntu.com/about/release-cycle), after a LTS release we would probably have a Standard Maintenance Period, where minor improvements, bug fixes, and security fixes would be released on an as-needed or periodic basis. Following that would be an Extended Maintenance Period where security fixes and critical bug fixes would be released on an as-needed basis. Following that the libspdm maintainers would consider the LTS release to be at end-of-life. However contributors would be free to provide pull requests against the LTS branch if they need to support it.

As far as the duration of support, Standard Maintenance could last for one year and Extended Maintenance could last for two to three year after that.

**Comment by jyao1** [comment]
Definition:
* Standard Maintenance: minor improvement + bug fix + security fix
* Extended Maintenance: critical bug fix + security fix
